_includes/getting-started.html

```diff
@@ -1,22 +1,111 @@
-<div class="anchor"><a name="getting-started"></a><h2>Run Meshery</h2></div>
-<div class="container flex">
+<html>
+  <head>
+    
+    <style>
+      
+      #bounce:hover{
+        -ms-transform: scale(1.5); /* IE 9 */
+  -webkit-transform: scale(1.5); /* Safari 3-8 */
+  transform: scale(1.5); 
+
+      }
+      @media screen and (max-width: 1900px){
+        #box{
+          width: 40%;
+          overflow: hidden;
+          margin-left: -60px;
+        }
+        #containers{
+          width: 80%;
+          position: relative;
+          
+          left: -100px;
+        }
+
+      }
+    
+      @media screen and (max-width: 1440px){
+        #box{
+          width: 40%;
+          overflow: hidden;
+          margin-left: -60px;
+        }
+        #containers{
+          width: 80%;
+          position: relative;
+          top: -100px;
+          left: -200px;
+        }
+
+      }
+      @media screen and (max-width: 1000px){
+        #box{
+          width: 100%;
+        }
+        #containers{
+          width: 100%;
+          height: 50%;
+          position: relative;
+          top: 300px;
+          left: -600px;
+        }
+
+      }
+
+      @media screen and (max-width: 500px){
+        #box{
+          width: 100%;
+        }
+        #containers{
+          width: 100%;
+          position: relative;
+          top: 300px;
+          left: -630px;
+        }
+
+      }
+
+     
+    </style>
```

Comment:
It would be good to pull there into an existing or new sass file - https://github.com/layer5io/meshery.io/tree/master/_sass

Reply:
@leecalcote sir, this style section for getting started section?

Reply:
@gunjanmimo right, yes. This document `getting-started.html` is a partial html snippet. If we can, it's best to land these new styles into an existing (or create a new) sass document.

Reply:
@leecalcote  sir, i have resolved this issue and added all the styling in respective sass file